src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/NodeFactory.cs

```diff
@@ -574,6 +574,8 @@ public void AttachToDependencyGraph(DependencyAnalysisFramework.DependencyAnalyz
             graph.AddRoot(ReadyToRunHeader, "ReadyToRunHeader is always generated");
             graph.AddRoot(new ModulesSectionNode(Target), "ModulesSection is always generated");
 
+            AddWellKnownTypes(graph);
```

Comment:
(There's also a duplicate of this list in `_wellKnownTypeNodes` - please consider finding a way to only have one list.)